test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/Search/CustomSearchParamTests.cs

```diff
@@ -86,7 +87,70 @@ public async Task GivenANewSearchParam_WhenReindexingComplete_ThenResourcesSearc
             await WaitForReindexStatus(reindexJobUri, "Completed");
 
             // When job complete, search for resources using new parameter
-            await ExecuteAndValidateBundle($"Patient?foo:exact={patientName}", expectedPatient.Resource);
+            await ExecuteAndValidateBundle($"Patient?fooCode:exact={patientName}", expectedPatient.Resource);
```

Comment:
>fooCode [](start = 53, length = 7)

nit: `{searchParam.Code}`  for consistency

Reply:
Updated.